packages/UI/src/components/organisms/PositionCard/index.ts

```diff
@@ -0,0 +1,10 @@
+export { default } from './PositionCard';
+
+export interface PositionCardProps {
+  // interaction: boolean;
+  // status: boolean;
+  interaction: 'all-h' | 'all-v' | 'save-only' | 'none';
+  style: 'default' | 'minimal' | 'mobile' | 'long';
+  buttonStyle: 'a' | 'b';
```

Comment:
@Nouri-Anouar 
Again not the best tokens since they don't tell me much as a consumer of the component.
we have to discuss this with Anna @annaisaacks

Reply:
makes sense! let's discuss it with Anna but what do you suggest?